docs/2.8.0/_toc.yml

```diff
@@ -1199,6 +1199,8 @@ subtrees:
               entries:
               - file: kubernetes/04-deploy-daml-enterprise/03-argocd-deployment/02-all-in-one-argocd-deployment
                 title: "Deploy Argo CD Applications"
+      - file: kubernetes/06-advanced-topics/high-availability
```

Comment:
We will probably have more pages under 'Advanced Topics', however, I didn't want to create another parent header for this one page.
Let me know what you think, i.e., whether we should put this page into a different place in the overall structure.